lib/modules/hosts/simplecove.js

```diff
@@ -0,0 +1,28 @@
+import { ajax } from '../../environment';
+
+export default {
+	moduleID: 'simplecove',
+	name: 'SimpleCove',
+	domains: ['simplecove.com'],
+	logo: 'https://simplecove.com/static/images/reslogo.jpg',
+	detect: ({ href }) => (/^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)).exec(href),
```

Comment:
Missing the end of the regex; should be: `(/^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)/i).exec(href),`